components/board/numato/board.go

```diff
@@ -384,6 +418,21 @@ func connect(ctx context.Context, name resource.Name, conf *Config, logger loggi
 		path = devs[0].Path
 	}
 
+	// Find the numato board's productid
+	products := getSerialDevices()
+
+	var productID int
+	for _, product := range products {
+		if product.ID.Vendor != 0x2a19 {
+			continue
+		}
+		productID = product.ID.Product
+	}
+
+	if productID != 0x805|0x802|0x800|0x0C05 {
```

Comment:
This is a confusing way to say `if productID != 0xC07` (the bitwise-or of the 4 values mentioned), which is probably not what you meant. This sucks, but I think you need to write it out:
```go
if productID != 0x800 && productID != 0x802 && productID != 0x805 && productID != 0xC05 {
```

(also, mild preference to sort the values, and either use leading 0's on all of them or on none of them. It makes it easier to check if a value you care about is mentioned in here.)